<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog</class>
 <widget class="QDialog" name="Dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>575</width>
    <height>325</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>575</width>
    <height>325</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>575</width>
    <height>325</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <property name="leftMargin">
    <number>0</number>
   </property>
   <property name="topMargin">
    <number>0</number>
   </property>
   <property name="rightMargin">
    <number>0</number>
   </property>
   <property name="bottomMargin">
    <number>0</number>
   </property>
   <property name="spacing">
    <number>0</number>
   </property>
   <item row="0" column="0">
    <widget class="QWidget" name="widget" native="true">
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <widget class="QTableWidget" name="tableWidget">
        <property name="font">
         <font>
          <family>Rockwell</family>
          <pointsize>12</pointsize>
         </font>
        </property>
        <property name="styleSheet">
         <string notr="true">color:dark-blue;
background-color: rgb(255, 255, 255);</string>
        </property>
        <column>
         <property name="text">
          <string>Level</string>
         </property>
         <property name="font">
          <font>
           <pointsize>20</pointsize>
          </font>
         </property>
         <property name="textAlignment">
          <set>AlignCenter</set>
         </property>
         <property name="background">
          <color>
           <red>0</red>
           <green>0</green>
           <blue>0</blue>
          </color>
         </property>
         <property name="foreground">
          <brush brushstyle="NoBrush">
           <color alpha="255">
            <red>255</red>
            <green>85</green>
            <blue>0</blue>
           </color>
          </brush>
         </property>
        </column>
       </widget>
      </item>
      <item>
       <widget class="QWidget" name="widget_2" native="true">
        <property name="styleSheet">
         <string notr="true">background-color: rgb(240, 240, 240);</string>
        </property>
        <layout class="QGridLayout" name="gridLayout_2">
         <item row="0" column="0">
          <widget class="QComboBox" name="DialogueAttribute">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="minimumSize">
            <size>
             <width>0</width>
             <height>30</height>
            </size>
           </property>
           <property name="font">
            <font>
             <family>Rockwell</family>
             <pointsize>12</pointsize>
            </font>
           </property>
           <property name="styleSheet">
            <string notr="true">background-color: rgba(255, 255, 255, 0.7);</string>
           </property>
           <item>
            <property name="text">
             <string>--Select Attribute--</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>Title</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>Price</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>Seller</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>Sold_Items</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>Country</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>Status</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>Seller_Rank</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>Shipping</string>
            </property>
           </item>
          </widget>
         </item>
         <item row="1" column="0">
          <widget class="QPushButton" name="add_Btn">
           <property name="minimumSize">
            <size>
             <width>165</width>
             <height>30</height>
            </size>
           </property>
           <property name="font">
            <font>
             <family>Rockwell</family>
             <pointsize>14</pointsize>
            </font>
           </property>
           <property name="styleSheet">
            <string notr="true">QPushButton{
	background-color:rgb(37,150,190);
	color: rgb(255, 255, 255);
	border-radius:10%;
}

QPushButton:hover{
	background-color: rgb(52, 205, 220 );
	color:rgb(0,0,0);
}</string>
           </property>
           <property name="text">
            <string>Add</string>
           </property>
          </widget>
         </item>
         <item row="2" column="0">
          <widget class="QPushButton" name="delete_Btn">
           <property name="minimumSize">
            <size>
             <width>165</width>
             <height>30</height>
            </size>
           </property>
           <property name="font">
            <font>
             <family>Rockwell</family>
             <pointsize>14</pointsize>
            </font>
           </property>
           <property name="styleSheet">
            <string notr="true">QPushButton{
	background-color:rgb(37,150,190);
	color: rgb(255, 255, 255);
	border-radius:10%;
}

QPushButton:hover{
	background-color: rgb(52, 205, 220 );
	color:rgb(0,0,0);
}</string>
           </property>
           <property name="text">
            <string>Delete</string>
           </property>
          </widget>
         </item>
         <item row="3" column="0">
          <widget class="QRadioButton" name="multi_ascending_Check">
           <property name="maximumSize">
            <size>
             <width>16777215</width>
             <height>25</height>
            </size>
           </property>
           <property name="font">
            <font>
             <family>Rockwell</family>
             <pointsize>11</pointsize>
            </font>
           </property>
           <property name="text">
            <string>Ascending</string>
           </property>
           <property name="checked">
            <bool>true</bool>
           </property>
          </widget>
         </item>
         <item row="4" column="0">
          <widget class="QRadioButton" name="multi_descending_Check">
           <property name="maximumSize">
            <size>
             <width>16777215</width>
             <height>25</height>
            </size>
           </property>
           <property name="font">
            <font>
             <family>Rockwell</family>
             <pointsize>11</pointsize>
            </font>
           </property>
           <property name="text">
            <string>Descending</string>
           </property>
          </widget>
         </item>
         <item row="5" column="0">
          <widget class="QWidget" name="widget_3" native="true">
           <property name="minimumSize">
            <size>
             <width>185</width>
             <height>70</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>16777215</width>
             <height>70</height>
            </size>
           </property>
           <layout class="QVBoxLayout" name="verticalLayout">
            <item>
             <widget class="QPushButton" name="multisort_Btn">
              <property name="minimumSize">
               <size>
                <width>165</width>
                <height>30</height>
               </size>
              </property>
              <property name="font">
               <font>
                <family>Rockwell</family>
                <pointsize>14</pointsize>
               </font>
              </property>
              <property name="styleSheet">
               <string notr="true">QPushButton{
	background-color:rgb(37,150,190);
	color: rgb(255, 255, 255);
	border-radius:10%;
}

QPushButton:hover{
	background-color: rgb(52, 205, 220 );
	color:rgb(0,0,0);
}</string>
              </property>
              <property name="text">
               <string>Sort Data</string>
              </property>
             </widget>
            </item>
           </layout>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
